lib/matplotlib/axes/_axes.py

```diff
@@ -7501,6 +7501,9 @@ def specgram(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
         -----
         The parameters *detrend* and *scale_by_freq* do only apply when *mode*
         is set to 'psd'.
+
+        The origin of the plot is forced to 'upper'. The image.origin rcParam
```

Comment:
I think this is more mysterious than helpful - most users won't know that specgram is implemented with `imshow`, and won't know what you are talking about wrt "origin".

Reply:
That is a good point. What about removing the first sentence: "The origin of the plot is forced to 'upper'."? That has the disadvantage that it is unclear what the spectrogram will look like before plotting it. 

The extended description of the function does explain that it calls imshow, and the **kwargs parameter documentation specifies that keyword arguments are passed on to imshow. Maybe it would be clearer to specify at the end of the note: "The keyword argument origin='upper' is automatically passed to '~.axes.Axes.imshow'".

Reply:
I wouldn't mention this in a note at all.  I'd just mention it above where you mention imshow and its kwargs already?